main.go

```diff
@@ -128,6 +157,41 @@ func runBenchmarks() (Set, error) {
 	return set, nil
 }
 
+func trimTagVersion(tagName string) string {
+	return strings.TrimLeft(tagName, tagVersionPrefix)
+}
+
+func getLatestRelease(repository *git.Repository) (prevVersionTag *plumbing.Reference, err error) {
+	var tagRefs storer.ReferenceIter
+	tagRefs, err = repository.Tags()
+	if err != nil {
+		return
+	}
+
+	type SemverTag struct {
+		Ref     *plumbing.Reference
+		Version semver.Version
+	}
+	tags := make([]SemverTag, 0)
+	err = tagRefs.ForEach(func(tagRef *plumbing.Reference) error {
+		tagName := tagRef.Name().Short()
+		v, err := semver.Make(trimTagVersion(tagName))
+		if err != nil {
+			klog.ErrorS(err, "Tag name is a not a valid semver, skipping", "tag", tagName)
```

Comment:
this should be `InfoS` IMO. Projects can define tags that are not semvers if they need them for other purposes.

Reply:
Done.

Reply:
Done.